geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloFeatureReader.scala

```diff
@@ -20,32 +20,24 @@ import org.geotools.data.{FeatureReader, Query}
 import org.locationtech.geomesa.core.index._
 import org.locationtech.geomesa.core.stats._
 import org.locationtech.geomesa.core.util.ExplainingConnectorCreator
+import org.locationtech.geomesa.feature.FeatureEncoding.FeatureEncoding
 import org.locationtech.geomesa.feature.SimpleFeatureEncoder
 import org.locationtech.geomesa.utils.stats.{MethodProfiling, TimingsImpl}
 import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 
 class AccumuloFeatureReader(dataStore: AccumuloDataStore,
                             query: Query,
-                            indexSchemaFmt: String,
                             sft: SimpleFeatureType,
-                            featureEncoder: SimpleFeatureEncoder)
+                            indexSchemaFmt: String,
```

Comment:
Probably for a later sprint . . . we need to pass around materialized ```IndexSchema```s rather than the format string to reduce the number of times we parse this string.